src/linear_regression_model/LinearRegressionModel.jsx

```diff
@@ -186,3 +136,43 @@ export class LinearRegressionModel extends Component {
         );
     }
 }
+
+const VerticalButton = ({flipGraphs, handleFlipGraphs, toggleLock}) => {
+    return (
+        <button
+            className={
+                flipGraphs ?
+                    'btn btn-light btn-sm' :
+                    'btn btn-light btn-sm active'}
```

Comment:
Noting that active / inactive states are barely distinguishable. Suggest making the states more visually distinct.

Reply:
Agreed, I'll bring it up in our next team meeting.